atlasdb-config/src/test/java/com/palantir/atlasdb/http/FailoverFeignTargetTest.java

```diff
@@ -146,6 +151,26 @@ public void blockingTimeoutExceptionResetsFailureCount() {
         }
     }
 
+    @Test
+    public void blockingTimeout() {
+        final FailoverFeignTarget spiedTarget = Mockito.spy(new FailoverFeignTarget<>(
+                SERVERS, 1, Object.class));
+        for (int i = 0; i < CLUSTER_SIZE; i++) {
+            spiedTarget.url();
+            spiedTarget.continueOrPropagate(EXCEPTION_WITH_RETRY_AFTER);
+        }
+
+        ArgumentCaptor<Long> argument = ArgumentCaptor.forClass(Long.class);
+        verify(spiedTarget, times(CLUSTER_SIZE)).pauseForBackoff(any(), argument.capture());
+
+        List<Long> arguments = argument.getAllValues();
+        assertThat(arguments.size()).isEqualTo(CLUSTER_SIZE);
+        for (int i = 0; i < CLUSTER_SIZE - 1; i++) {
```

Comment:
If we specify a cluster size for this test of 3 (i.e. without using the CLUSTER_SIZE constant), we can write a single assert to verify that `arguments.containsExactly(1L, 1L, 1000L)` (pseudo-test-code, actual method required may be different).